packaging/torcharrow/meta.yaml

```diff
@@ -24,16 +24,15 @@ requirements:
     - cffi
     - glog==0.4.0
     - libsodium
-    {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
+    - pytorch
+    # {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
 
 build:
   string: py{{py}}
   script_env:
     - BUILD_VERSION
     - CPU_TARGET
 
-#GitHub Actions usually provide Mac without AVX2 support, for now just skip the test until Velox can be run without AVX2
```

Comment:
so the test is already enabled now, right?

Reply:
You might need to add command `pytest --no-header -v torcharrow/test` back the test